io.github.ec_.Quake3e.OpenArena.appdata.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014 Richard Hughes <richard@hughsie.com> -->
+<application>
+  <id type="desktop">io.github.ec_.Quake3e.OpenArena</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <provides>
+    <id>io.github.ec_.Quake3e.OpenArena.desktop</id>
+  </provides>
+  <name>OpenArena (Quake3e)</name>
+  <summary>Free and open source first-person shooter</summary>
+  <description>
+    <p>
+      OpenArena is a free and open source first-person shooter game based
+      heavily on the Quake III Arena-style deathmatch.
+      Score frags to win the game using a balanced set of weapons each designed
+      for different situations.
+      Guns include a chaingun, rocket launcher, shotgun, and railgun.
+    </p>
+    <p>
+      OpenArena's style of play is very fast and requires skill to be played
+      successfully online.
+      Gameplay modes include Deathmatch, Team Deathmatch, Tournament, Capture
+      the Flag, Elimination, CTF Elimination, Last Man Standing and Double
+      Domination.
+    </p>
+    <p>
+      This version of OpenArena is based on the Quake3e engine by "Ec" on GitHub, I do not take credit for his work or any of the work done by the OpenArena team.
+      This FlatPak was made to be a replacement of the original one released on 2012, as it contains bugs and annoyances not present in this engine. The gameplay is 100% compatible and if not, please make an issue in my repo(xplshn/openarena), I'll investigate it.
+    </p>
+  </description>
+  <url type="homepage">http://www.openarena.ws/</url>
+  <url type="source">https://xplshn.github.io/</url>
```

Comment:
This isn't right.

Reply:
Couldn't find comprehensible documentation on how to add a repository link, is it not possible?

Reply:
I'll change it right now

Reply:
Fixed